.github/workflows/main.yml

```diff
@@ -19,10 +19,7 @@ jobs:
           - x25519_u64,ristretto255_u64
         toolchain:
           - stable
-          - 1.51.0
-        exclude:
-          - backend_feature: p256
-            toolchain: 1.51.0
+          - 1.56.1
```

Comment:
Should this be 1.56.0 to keep consistent?

Reply:
1.56.1 is an important security patch: https://github.com/rust-lang/rust/releases/tag/1.56.1. Considering that there are no other changes I think that's alright?

Reply:
I see. Should lines 102 and 125 also be updated to 1.56.1 then as well (instead of 1.56.0)?

Reply:
Indeed, I missed that. Done!